### ISSUE TITLE
Support show localtime rather than utc?

### ISSUE THREAD

**Issue by wrvsrx** [ad-si/TaskLite#82]
It's inconvinient to always use utc time to edit task. Can tasklite improve UX by supporting specify time using local time rather than utc?

**Comment by ad-si** [comment]
Yeah, definitely a good idea. I didn't prioritize this yet, as I've all my clocks set to UTC and so it's not relevant for me. Also, I'm using https://github.com/ad-si/tu for specifying times, so I don't need to know the absolute value, but can simply use relative values.

Any help to implement it would be very appreciate! If you're interested, there is a little bit of documentation available: https://tasklite.org/development

**Comment by wrvsrx** [comment]
I will try to add this feature when I have time.

**Comment by gour** [comment]
> Yeah, definitely a good idea. I didn't prioritize this yet, as I've all my clocks set to UTC and so it's not relevant for me. Also, I'm using https://github.com/ad-si/tu for specifying times, so I don't need to know the absolute value, but can simply use relative values.

I've installed `tu` but still have to specify specific time which could be, maybe, more convenient with the ability to specify concrete time expressed in my local timezone.

**Comment by gour** [comment]
I'm currently migrating my TW database into Tasklite, The experience is very pleasant, but I was bitten with the UTC vs local time and have to rework them.

Besides using `tu`, I have e.g. schedule for garbage collection which has to be prepared by 7:00 AM local time and if I'm late someone will be angry at me. 😄 

So, I'd **highly** appreciate  the option to have an ability to specify absolute time (e.g. when not using `tu˙) as well as have Tasklite enable me to show tasks with the specified local time. 🙏
